### ISSUE TITLE
Convolutional layer of Complex-valued sample 

### ISSUE THREAD

**Issue by Ahmedest61** [EricAlcaide/jscc_tensorflow#1]
Many thanks for your work employing DL for wireless communication. I am implementing your paper in Keras (tensorflow) and need to confirm about the complex-value channel (last convolutional layer of encoder before normalization & first layer of decoder); precisely have you used "complexnn.conv.ComplexConv2D"? Could you also please confirm me about the other convolutional layers of encoder and decoders, are those "tensorflow.keras.layers.Conv2D" or "complexnn.conv.ComplexConv2D". Thanks you.
